test/globalHooks.js

```diff
@@ -9,7 +10,7 @@ before('global setup', () => {
 
 after('Check all nocks have been called', () => {
   try {
-    console.log(nock.pendingMocks());
+    log.fatal(nock.pendingMocks());
```

Comment:
Logged as `fatal` as the tests run with `NODE_ENV` where only `fatal` logs are output